commands/stats.js

```diff
@@ -0,0 +1,77 @@
+import * as bungie from '../bungieApi';
+import * as util from 'util';
+
+export const command = '/stats';
+export async function handler(payload) {
+  // do some stuff with the payload
+
+  const name = payload.text.length ? payload.text : payload.user_name;
+  const msg = {
+    "response_type": "ephemeral",
+    attachments: []
+  };
+
+  const member = [
+    ...await bungie.getPlayerID(1, name),
+    ...await bungie.getPlayerID(2, name)
+  ];
```

Comment:
This pattern will do each request serially, when you probably wanted them in parallel. In addition, if `getPlayerID` ever returns anything other than an Array, this will throw.

Alternative:

```javascript
const member = (await Promise.all([
  bungie.getPlayerID(1, name),
  bungie.getPlayerID(2, name),
])).reduce((accum, value) => {
  if (value) {
    return accum.concat(value);
  }
  return accum;
}, []);
```

You could make this even shorter by using lodash:

```javascript
// at the top:
import { flatten, compact } from 'lodash';

// ...

const member = flatten(compact(Promise.all([
  bungie.getPlayerID(1, name),
  bungie.getPlayerID(2, name),
]);
```